src/components/UserPreferencesPanel/CategoryPreferences.tsx

```diff
@@ -5,6 +5,7 @@ import ToggleInput from './ToggleInput';
 
 interface Props {
   category: string;
+  channels?: string[];
```

Comment:
Do you see a downside if we were to type this as a union?

```suggestion
  channels?: Array<'inApp' | 'email' | 'webPush' | 'mobilePush'>;
```

Reply:
I wasn't aware you could do that (haven't done a lot of TypeScript).
The only downside I can think of is the case where MagicBell adds another channel, and this component hasn't been updated yet. As it is, the channel would appear, but the wrong class would be applied (probably not a big deal).
Would this more restrictive type cause an error in that scenario?

Reply:
Yes, it would cause an error because it's "unknown" at that moment. I think it's worth it though. The number of channels is limited, while a simple typo ("webpush" instead of "webPush") is easily made. So let's make this more strict.

Reply:
If someone's using this component on its own and manually typing in these channel names (not pulling them from settings), that seems like an anti-pattern anyway. But I don't consider this my call to make, so I'll commit your suggestion.

Reply:
For the record, with the addition of the Slack delivery channel, this component is already out of date, and because of the union type, SupportBee's UI is wrong:
<img width="529" alt="image" src="https://user-images.githubusercontent.com/11638917/153302539-10e5f5db-a45e-431f-870c-338a735d1455.png">